src/Libraries/Microsoft.Extensions.AI.Abstractions/HostedMcpServerToolAlwaysRequireApprovalMode.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Indicates that approval is always required for tool calls to a hosted MCP server.
+/// </summary>
+/// <remarks>
+/// Use <see cref="HostedMcpServerToolApprovalMode.AlwaysRequire"/> to get an instance of <see cref="HostedMcpServerToolAlwaysRequireApprovalMode"/>.
+/// </remarks>
+[DebuggerDisplay(nameof(AlwaysRequire))]
+public sealed class HostedMcpServerToolAlwaysRequireApprovalMode : HostedMcpServerToolApprovalMode
+{
```

Comment:
We should give this an explicit public parameterless ctor so that we can add XML comments to it. I believe elsewhere in such cases those XML comments discourage direct use of the constructor and instead point folks to use the singleton property.